docs/source/projects/user-interfaces.rst

```diff
@@ -8,12 +8,29 @@ performed using a wide range of programming languages.
 
 .. glossary::
 
+    `JupyterLab <https://jupyterlab.readthedocs.io/en/latest/>`__
+        Advanced Web-based application for authoring documents that combine live-code
+        with narrative text, equations and visualizations.
+        `Documentation <https://jupyterlab.readthedocs.io/en/latest/>`__ |
+        `Repo <https://github.com/jupyterlab/jupyterlab>`__
+
     `Jupyter Notebook <https://jupyter-notebook.readthedocs.io/en/latest/>`__
-        Web-based application for authoring documents that combine live-code
+        Simplified web-based application for authoring documents that combine live-code
```

Comment:
```suggestion
        A web-based application for authoring documents that combine live-code
```

I would not call it simplified. [nteract](https://nteract.io/) was simplified, Jupyter Notebook is _the_ reference point for many.